pyaedt/modeler/Primitives.py

```diff
@@ -1244,20 +1248,21 @@ def create_polyline(self, position_list, segment_type=None,
             Height of the cross-section for type ``"Rectangle"`` or ``"Isosceles 
             Trapezoid"`` only. The default is ``1``.
         xsection_num_seg : int, optional
-            Number of segments in the cross-section surface for types ``"Circle"``, 
+            Number of segments in the cross-section surface for type ``"Circle"``, 
             ``"Rectangle"``, or ``"Isosceles Trapezoid"``. The default is ``0``. The
             value must be ``0`` or greater than ``2``.
         xsection_bend_type : str, optional
-            Type of the bend for the cross-section. The default is ``None``, which sets
-            the bend type to ``"Corner"``. For the type ``"Circle"``, the bend type 
+            Type of the bend for the cross-section. The default is ``None``, in which 
+            case the bend type is set to ``"Corner"``. For the type ``"Circle"``, the bend type 
             should be set to ``"Curved"``.
         
         Returns
         -------
         Polyline
-            Object with additional methods for manipulating the polyline.  For example,
-            ``insert_segment``. The object ID of the created polyline can be accessed
-            via ``Polyline.id``.
+            Object with additional methods for manipulating the polyline.  
+            
+            For example, `insert_segment`. The object ID of the created polyline can be accessed
+            via `Polyline.id`.
```

Comment:
```suggestion
            via ``Polyline.id``.
```
Single backtick since we're not calling out class and this should be literal.